readalongs/text/util.py

```diff
@@ -295,3 +298,68 @@ def parse_time(time_string: str) -> int:
         raise ValueError(
             f'cannot parse "{time_string}" as a valid time in h/m/s/ms'
         ) from e
+
+
+TEMPLATE_README_TXT = """
```

Comment:
Let's have a comment here that says this text should be kept in sync with the same text in the Web-Component repo, with a pointer to where that is. And let's add a comment there pointing here. That way we'll know to update both when we update one.

Reply:
DONE